services/pub/pub-publisher.tester.js

```diff
@@ -0,0 +1,26 @@
+import { ServiceTester } from '../tester.js'
+export const t = new ServiceTester({
+  id: 'PubPublisher',
+  title: 'Pub Publisher',
+  pathPrefix: '/pub',
+})
+
+t.create('package publisher').get('/publisher/path.json').expectBadge({
```

Comment:
This accompanies the import/registration change

```suggestion
t.create('package publisher').get('/path.json').expectBadge({
```